### ISSUE TITLE
(feat: addons Karpenter) support consolidation in Karpenter 0.15

### ISSUE THREAD

**Issue by javydekoning** [aws-quickstart/cdk-eks-blueprints#486]
### Describe the feature

Karpenter 0.15 introduces a new configuration. 

```yaml
spec:
  # Enables consolidation which attempts to reduce cluster cost by both removing un-needed nodes and down-sizing those
  # that can't be removed.  Mutually exclusive with the ttlSecondsAfterEmpty parameter.
  consolidation:
    enabled: true
```

This needs to be added to https://github.com/aws-quickstart/cdk-eks-blueprints/blob/f7fba04f2488556366779e56ebb88d860747aa5a/lib/addons/karpenter/index.ts

I suggest we default to `True` if not provided. 

### Use Case

consolidation which attempts to reduce cluster cost by both removing un-needed nodes and down-sizing those that can't be removed.  Mutually exclusive with the ttlSecondsAfterEmpty parameter.

### Proposed Solution

Something like: 

```ts
consolidation?: {
    'enabled': boolean,
}

const consolidation = this.options.consolidation || { consolidation: { enabled: true }};
```

### Acknowledgements

- [x] I may be able to implement this feature request

### CDK version used

N/A

### EKS Blueprints Version

1.2.0

### Node.js Version

N/A

### Environment details (OS name and version, etc.)

N/A

**Comment by youngjeong46** [comment]
@javydekoning thanks for the submission. We haven't up-ed our support of Karpenter to 0.15, so it will be in a future PR with some of the other ongoing Karpenter issues (your others included.)

**Comment by youngjeong46** [comment]
I have a PR in flight that takes care of this issue (and upgrade to 0.16 support). Your other issue, as I mentioned in the ticket, will have to be a followup due to breaking changes.

**Comment by youngjeong46** [comment]
Addressed (and upgraded to 0.16 support) Per PR #489. Closing the issue.
